stix_shifter_modules/reaqta/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -1423,9 +1423,9 @@
         "key": "x-reaqta-event.filter_name",
         "object": "x-reaqta"
       },
-      "fsName": {
-        "key": "file.extensions.x-reaqta-data.fsname",
-        "object": "file_data"
+      "filename": {
+        "key": "process.name",
+        "object": "process"
```

Comment:
there's no process.name in stix2.1. so revert back the key and objects for filename